lib/active_model/serializer.rb

```diff
@@ -37,6 +37,7 @@ class << self
       attr_accessor :_cache_except
       attr_accessor :_cache_options
       attr_accessor :_cache_digest
+      attr_accessor :_type
```

Comment:
n/m, it can/should be removed since `class_attribute :_type, instance_writer: false` is added below